src/pages/ArticlePage.jsx

```diff
@@ -0,0 +1,37 @@
+import { useEffect, useState } from "react";
+import { geArticleById } from "../api";
+import { useParams } from "react-router-dom";
+
+export const ArticlePage = () => {
+  const { article_id } = useParams();
+  const [article, setArticle] = useState({});
+
+  useEffect(() => {
+    geArticleById(article_id).then((response) => {
+      setArticle(response);
+    });
+  }, [article_id]);
```

Comment:
You can still have a loading here. If you want to, instead of repeating the same logic for loading you could extract it into it's own Loading component!